packages/create-webiny-project/init.js

```diff
@@ -238,19 +237,19 @@ module.exports = async function({ root, appName, templateName, tag, log }) {
                 try {
                     await require(templatePath)({ appName, root });
                 } catch (err) {
-                    console.log(err);
                     throw new Error("Unable to perform template-specific actions.", err);
                 }
             }
         }
     ]);
 
-    try {
-        await tasks.run();
-    } catch (err) {
-        console.error("Unable to complete project initialization", err);
-        return;
-    }
+    await tasks.run().catch(async err => {
+        console.error(err);
+        console.log("\nCleaning up project...");
+        await execa("rm", ["-r", root]);
```

Comment:
Will `rm` work  in Git Bash on Windows, without WSL?
I think we should use a tool like [rimraf](https://www.npmjs.com/package/rimraf) here, which is cross-platform.

Reply:
I wanted to confirm this so I tested it out on different shells:
rm works on my powershell, and out of the box with git bash (You can configure when you install Git Bash so that you do get this feature for powershell I believe)
I also found this which points out some flaws in rimraf:
http://www.ashleysheridan.co.uk/blog/Why+You+Should+Not+Use+Rimraf
Though the fact its a bit slow isn't a big deal for the small size of the project.
I also added the bubbling up of errors so the output log has proper outputs.
I added some help text for the log flag, swapped it to use strings instead and accepts path or file names.

Reply:
Great! 👍